Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # builder image
-FROM golang as builder
+FROM golang:1.12.4 as builder
```

Comment:
Why pinning to `1.12.4`? We have no specific requirements to have this specific version right now.

Reply:
IMO a tag should be specified. Otherwise local builds may use whatever "latest" has been pulled down to the machine. In my case, this was a old version of `golang:latest` which caused build failures.

Reply:
Sounds legit, we just need to remember to update this, but I guess that's doable 😉

Reply:
👍